corehq/apps/app_manager/templates/app_manager/summary_new.html

```diff
@@ -6,6 +6,15 @@
 {% load compress %}
 {% load djangular_tags %}
 
+{% block stylesheets %}
+{% compress css %}
```

Comment:
you are correct! I mistakenly thought this was inheritinge from hqwebapp/base.html